package.json

```diff
@@ -29,6 +29,7 @@
     "dev-api": "./scripts/build/copy-static-files.sh && (npm run --prefix shared-libs/cht-datasource build-watch & npm run --prefix api run-watch)",
     "dev-sentinel": "npm run --prefix shared-libs/cht-datasource build-watch & npm run --prefix sentinel run-watch",
     "local-images": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && node scripts/build/cli localDockerComposeFiles",
+    "set-local-version": "export VERSION=${VERSION:-$(git branch --show-current)}",
```

Comment:
This is a duplicate of what the versions script is doing. Where should this be used? Is this needed?

Reply:
It was a mistake, I was testing other options before, I will remove it, thanks!